src/test/java/io/confluent/kafka/connect/datagen/DatagenTaskTest.java

```diff
@@ -201,6 +203,14 @@ public void shouldFailToGenerateMoreRecordsThanSpecified() throws Exception {
     }
   }
 
+  @Test(expected = ConnectException.class)
+  public void shouldFailIfSchemaKeyFieldNotPresentInSchema() throws Exception {
+    config.put(DatagenConnectorConfig.QUICKSTART_CONF, DatagenTask.Quickstart.USERS.name().toLowerCase(Locale.getDefault()));
```

Comment:
It looks like quickstart names [should be case-insensitive](https://github.com/confluentinc/kafka-connect-datagen/blob/c661b0d76eab491d2d9dfe6a1482c65141caaea6/src/main/java/io/confluent/kafka/connect/datagen/DatagenTask.java#L147); can we remove the `toLowerCase` invocation here and make things a little cleaner?